### PR TITLE
You've made your last delivery, kid.

### PR DESCRIPTION
**Pull request by PlayerAdelheid**
From where you're kneeling it must seem like an 18-carat run of bad luck...

### Executions and Suicides
-It takes 4 uninterrupted seconds to begin an execution (Help intent mouth)
-Reflavored to have Fallout: NV referenced descriptions

### CODE REVIEW

**Comment by JasmineRickards** [comment]
Execution time needs to be a bit longer, and the 10x multiplier makes it too good to use in combat on crit enemies.

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -902,7 +902,7 @@ ATTACHMENTS
 	playsound('sound/weapons/dink.ogg', 30, 1)
 
 	if(chambered && chambered.BB)
-		chambered.BB.damage *= 5
+		chambered.BB.damage *= 10
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/gun.dm:905]:
```suggestion
		chambered.BB.damage *= 5
```

File: code/modules/projectiles/gun.dm [hunk 2]
```diff
@@ -879,14 +879,14 @@ ATTACHMENTS
...
 	else
-		target.visible_message("<span class='warning'>[user] points [src] at [target]'s head, ready to pull the trigger...</span>", \
-			"<span class='userdanger'>[user] points [src] at your head, ready to pull the trigger...</span>")
+		target.visible_message("<span class='warning'>[user] puts the barrel of [src] against [target]'s forehead. The game was rigged from the start.</span>", \
+			"<span class='userdanger'>[user] puts the barrel of [src] to your head. This must be an 18-carat run of bad luck...</span>")
 
 	busy_action = TRUE
 
-	if(!bypass_timer && (!do_mob(user, target, 120) || user.zone_selected != BODY_ZONE_PRECISE_MOUTH))
+	if(!bypass_timer && (!do_mob(user, target, 15) || user.zone_selected != BODY_ZONE_PRECISE_MOUTH))
```
Comment (PlayerAdelheid) [inline code/modules/projectiles/gun.dm:889]:
```suggestion
	if(!bypass_timer && (!do_mob(user, target, 30) || user.zone_selected != BODY_ZONE_PRECISE_MOUTH))
```

**Comment by PlayerAdelheid** [comment]
> Execution time needs to be a bit longer, and the 10x multiplier makes it too good to use in combat on crit enemies.

Made it 4 seconds and returned to 5

**Pull request merged**